src/components/Personalization/PersonalizationModal.jsx

```diff
@@ -44,7 +97,35 @@ export const PersonalizationModal = ({
         </div>
       </div>
 
-      <Wallpaper client={client} />
+      <div
+        style={{
+          width: '100%',
+          height: '100%',
+          position: 'relative'
```

Comment:
no inline style please. At least not this way because if the `PersonalizationModal` re-render, it'll create a new style object, and then it'll re-render the children here